src/CernerOneIcons-CSV.csv

```diff
@@ -80,7 +80,7 @@ doNotDisturb,src/icons/0039_minus_i_heavyCircleRed.svg,,
 close,src/icons/0041_x_a.svg,themeable ,
 clear,src/icons/0041_x_a.svg,themeable ,
 incomplete,src/icons/0041_x_i_heavyCircle.svg,themeable ,
-incomplete_low_light,src/icons/LL_0041_x_i_heavyCircle.svg,,
+incomplete_low_light,src/icons/LL_0041_x_i_heavyCircleWhite.svg,,

```

Comment:
Not sure if this is the right file. Currently, the CSV file points to `src/icons/LL_0041_x_i_heavyCircle.svg`, but that file doesn't exist. I've updated CSV to poing towards `src/icons/LL_0041_x_i_heavyCircleWhite.svg`, but maybe the `LL_0041_x_i_heavyCircle.svg` file was correct and the SVG just needs to be added?

Reply:
Ended up removing incomplete_low_light altogether as it wasn't needed.

Reply:
Ah ok, I've removed this entry from the CSV file.